services/user-management/src/main/java/com/foodie/user/usermanagement/UserManagementApplication.java

```diff
@@ -0,0 +1,71 @@
+package com.foodie.user.usermanagement;
+
+import static com.foodie.models.enums.Role.*;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+import com.foodie.jwt.AuthenticationService;
+import com.foodie.models.contracts.RegisterRequest;
+
+@SpringBootApplication()
+@ComponentScan({"com.foodie"})
```

Comment:
Without this it is not creating my models on the db.

Reply:
Then you should check your entity classes definition, chances are you're missing something and if you don't figure it out now, it might come back to hunt you later. For one I wouldn't call my table User, because user is a keyword in many database engines, and you might come across a case where one of them will do something stupid because of it

Reply:
table is using @table annotation and been called users